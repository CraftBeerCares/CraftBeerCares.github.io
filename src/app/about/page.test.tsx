import { render } from "@testing-library/react";
import Page from "./page";

it("renders About page", () => {
  const { container } = render(<Page />);
  expect(container).toMatchSnapshot();
});
