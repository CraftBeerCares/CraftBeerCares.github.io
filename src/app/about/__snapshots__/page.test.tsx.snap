// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders About page 1`] = `
<div>
  <div
    class="pageContainer"
  >
    <div
      class="header"
    >
      <header>
        <div
          class="header"
        >
          <button
            class="burgerMenuButton"
          >
            <img
              alt="Burger Menu"
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              src="/assets/menu.svg"
              style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: contain; color: transparent; filter: invert(1);"
            />
          </button>
          <h1
            class="className title"
          >
            Craft Beer Cares
          </h1>
          <nav
            class="topNav className"
          >
            <a
              class="navLink"
              href="/"
            >
              Home
            </a>
            <a
              class="navLink"
              href="/about"
            >
              About
            </a>
            <a
              class="navLink"
              href="/join_us"
            >
              Join Us
            </a>
            <a
              class="navLink"
              href="/events"
            >
              Events
            </a>
            <a
              class="navLink"
              href="/shop"
            >
              Shop
            </a>
            <a
              class="navLink"
              href="/contact"
            >
              Contact
            </a>
          </nav>
        </div>
      </header>
    </div>
    <div
      class="contentContainer"
    >
      <main
        class="className mainContent"
      >
        <div
          class="imageWithBlurb"
        >
          <div
            class="imageWrapper"
          >
            <img
              alt="A picture of me"
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              sizes="100vw"
              src="/_next/image?url=%2Fassets%2F003.webp&w=3840&q=75"
              srcset="/_next/image?url=%2Fassets%2F003.webp&w=640&q=75 640w, /_next/image?url=%2Fassets%2F003.webp&w=750&q=75 750w, /_next/image?url=%2Fassets%2F003.webp&w=828&q=75 828w, /_next/image?url=%2Fassets%2F003.webp&w=1080&q=75 1080w, /_next/image?url=%2Fassets%2F003.webp&w=1200&q=75 1200w, /_next/image?url=%2Fassets%2F003.webp&w=1920&q=75 1920w, /_next/image?url=%2Fassets%2F003.webp&w=2048&q=75 2048w, /_next/image?url=%2Fassets%2F003.webp&w=3840&q=75 3840w"
              style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: cover; color: transparent;"
            />
          </div>
          <article
            class="article"
          >
            <h1
              class="title className"
            >
              The Craft Beer Cares Story
            </h1>
            <p>
              Craft Beer Cares was founded by Gautam Bhatnagar and Michaela Zelenanska in 2017. Inspired by other initiatives such as Cook for Syria that were happening at the time, the two took their social interests and contacts in craft beer and worked towards organising something that could raise money for great causes.
            </p>
            <br />
            <p>
              Our first festival took place at BrewClub in Clapton, 1-2 July 2017. Since then, thanks to many friends and contacts within the craft beer industry as well as other wonderful friends, family members, volunteers, and enthusiastic attendees, we have held a grand total of 4 beer festivals within 4 years, and have raised over Â£50,000 for charitable organisations.
            </p>
          </article>
        </div>
      </main>
      <footer
        class="footer"
      >
        <div
          class="footerMobile"
        >
          <div
            class="footerLogo"
          >
            <img
              alt="Craft Beer Cares Logo"
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              src="/assets/logo_bw.svg"
              style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent; filter: invert(1);"
            />
          </div>
        </div>
        <div
          class="footerDesktop"
        >
          <div
            class="footerColumn"
          >
            <h2
              class="className title"
            >
              Craft Beer Cares
            </h2>
            <nav
              class="className footerLinks"
            >
              <a
                class="navLink"
                href="/"
              >
                Home
              </a>
              <a
                class="navLink"
                href="/about"
              >
                About
              </a>
              <a
                class="navLink"
                href="/join_us"
              >
                Join Us
              </a>
              <a
                class="navLink"
                href="/events"
              >
                Events
              </a>
              <a
                class="navLink"
                href="/shop"
              >
                Shop
              </a>
              <a
                class="navLink"
                href="/contact"
              >
                Contact
              </a>
            </nav>
            <div
              class="socialLinks"
            >
              <a
                class="socialLink"
                href="https://www.facebook.com/craftbeercares"
              >
                <img
                  alt="Facebook"
                  data-nimg="1"
                  decoding="async"
                  height="30"
                  loading="lazy"
                  src="/assets/facebook.svg"
                  style="color: transparent;"
                  width="30"
                />
              </a>
              <a
                class="socialLink"
                href="https://www.instagram.com/craftbeercares/"
              >
                <img
                  alt="Instagram"
                  data-nimg="1"
                  decoding="async"
                  height="30"
                  loading="lazy"
                  src="/assets/instagram.svg"
                  style="color: transparent;"
                  width="30"
                />
              </a>
              <a
                class="socialLink"
                href="https://twitter.com/craftbeercares"
              >
                <img
                  alt="Twitter"
                  data-nimg="1"
                  decoding="async"
                  height="30"
                  loading="lazy"
                  src="/assets/twitter.svg"
                  style="color: transparent;"
                  width="30"
                />
              </a>
            </div>
            <div
              class="className footerQuote"
            >
              Craft Beer Cares - Drinking beer for good Stuff since 2017
            </div>
          </div>
          <div
            class="footerColumn"
          >
            <div
              class="footerLogo"
            >
              <img
                alt="Craft Beer Cares Logo"
                data-nimg="fill"
                decoding="async"
                loading="lazy"
                src="/assets/logo_bw.svg"
                style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent; filter: invert(1);"
              />
            </div>
          </div>
          <div
            class="footerColumn"
          >
            Right
          </div>
        </div>
      </footer>
    </div>
  </div>
</div>
`;
